crates/core/txhash/Cargo.toml

```diff
@@ -10,3 +10,4 @@ hex = {workspace = true}
 penumbra-proto = {workspace = true, default-features = false}
 penumbra-tct = {workspace = true, default-features = true}
 serde = {workspace = true}
+getrandom = {workspace = true}
```

Comment:
should be added as a dev dependency?

Reply:
yeah should've made this a dev dependency

Reply:
reverting back, seems to fail as a dev dependency